<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPrincipal</class>
 <widget class="QMainWindow" name="vtnPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>429</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>Ingrese solo números</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlimpiar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>NOMBRE ESTUDIANTE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbsemestre">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Seleccionar</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1er</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2do</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3er</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4to</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5to</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6to</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7mo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8vo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9no</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10mo</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>APELLIDO ESTUDIANTE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcedula">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ingrese solo números</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>CEDULA ESTUDIANTE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtModaEstud">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="placeholderText">
     <string>Presencial, virtual e hibrida</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNombre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbApellido">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCedula">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFacultad">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Facultad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbModaEstud">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modalidad de estudio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSemestre">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Semestre:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btneliminar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnactualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnbuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btninsertar">
       <property name="text">
        <string>Insertar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="cbfacultad">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Arquitectura y Urbanismo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Administrativas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Agrarias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Económicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Matemáticas y Físicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Médicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Psicológicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Químicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Comunicación Social</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Educación Física,
Deportes y Recreación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Filosofía,
Letras y Ciencias de la
Educación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ingeniería
Industrial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ingeniería
Química</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad deJurisprudenciaycienicasSocialesyPolíticas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Medicina
Veterinaria y
Zootecnia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad Piloto de
Odontología</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facultad de Ciencias Naturales

</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtCarrera">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>INGRESE SU CARRERA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCarrera">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEdad">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spEdad">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtcedula</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtApellido</tabstop>
  <tabstop>cbfacultad</tabstop>
  <tabstop>txtModaEstud</tabstop>
  <tabstop>cbsemestre</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnlimpiar</tabstop>
  <tabstop>btninsertar</tabstop>
  <tabstop>btnbuscar</tabstop>
  <tabstop>btnactualizar</tabstop>
  <tabstop>btneliminar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
